mods/aether1.json

```diff
@@ -4,6 +4,34 @@
     "authors": ["Kingbdogz", "Kodaichi", "Shockah", "Flan", "303"],
     "versions": [
         {
+            "name": "1.2.5",
+            "desc": "Forge and Player API required",
+            "mcvsn": ["1.2.5"],
+            "files": [
+                {
+                    "filename": "Aether_1.9_Pre-Release.zip",
+                    "ipfs": "QmXa9Fa99wTFhr4oNSrFG9GY3cnTesUqoCMhSBQMYBg33q",
+                    "archived": "28b7f6dfd6093ae7353fb88428f24ab5affbf4694d55825d5ea2337ef2d89e2d/Aether_1.9_Pre-Release.zip",
```

Comment:
As I mentioned in my earlier comment, please don't put `archived` fields in mods you're adding. I'll add them myself when we upload the files to S3.